pipeline/test/test_pipeline.py

```diff
@@ -159,7 +197,6 @@ def test_scatter_gather(self):
         self.assertEqual(self.read(output_file), '2\n1\n0')
         self.rm(output_file)
 
-
     # FIXME: Run docker inside docker
```

Comment:
what's the deal with this one?

Reply:
You can't do docker run in k8s.

Reply:
let's delete this and add an issue describing it.